yt/data_objects/tests/test_dataset_access.py

```diff
@@ -24,9 +29,9 @@ def test_box_creation():
     region = ds.region(center, left_edge, right_edge)
 
     for b in boxes:
-        assert_equal(b.left_edge, region.left_edge)
-        assert_equal(b.right_edge, region.right_edge)
-        assert_equal(b.center, region.center)
+        assert_almost_equal(b.left_edge, region.left_edge)
```

Comment:
Why did are these now `almost`?

Reply:
I think there's an extra floating point operation happening somewhere that introduced some roundoff error.